### DISCUSSION TITLE
Multi-plan projects?

### DISCUSSION THREAD

**Discussion by saward** [theory/sqitch#652] [Q&A]
Hi,

`sqitch add` documentation mentions support for multiple plans, but I'm not sure exactly how I should be structuring my project to allow multiple plans.

My goal is to have some deploy/test/revert scripts that are present in every plan, and some that are only in specific plans.  This corresponds to tables that are shared across all projects vs tables that are only specific to a particular project.  I could create separate folders with separate sqitch.conf files, but pointing to the same folder with migrations, but I'm not confident this is the intended way to handle multiple plans.

I'd love to have some advice on the intended way to do this.

Thanks!

**Comment by theory** [comment]
I suggest you start by reading [sqitch-configuration](http://sqitch.org/docs/manual/sqitch-configuration/) if you haven't already. It covers a number of different plan and engine use-cases. Note that the configuration is hierarchical, so you can have a top-level `sqitch.conf` file, and then additional `sqitch.conf` files in each directory with configuration specific to that directory.
